Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics.MicroLayout/Driver/Controls/Circle.cs

```diff
@@ -1,17 +1,13 @@
 ﻿namespace Meadow.Foundation.Graphics.MicroLayout;
 
-public struct Coordinate2D
-{
-    public int X { get; set; }
-    public int Y { get; set; }
-}
-
 /// <summary>
 /// Represents a circle in the user interface.
 /// </summary>
 public class Circle : ThemedControl
```

Comment:
I'm wondering now if circle makes sense as a control ... we should talk about how we should handle drawing primitives

Reply:
I would have preferred an Elipse, but we don't have the ability to draw that.  It matches with the Box control.  But yes, we can definitely discuss.

Reply:
nothing to block on of course